src/api/public/apidocs-new/paths/search_owner.yaml

```diff
@@ -1,32 +1,40 @@
 get:
-  summary: Search for the default responsible person or group (the owner).
+  summary: Search for the users or groups who are the owners of a project or package.
   description: |
-    Search for the default responsible person or group (the owner) by a user, group
-    or binary name.
+    The owners are users/groups with the maintainer or bugowner roles in the given project/package.
+
+    The owner search will get results only if one of the ancestors or related projects of the searched
+    package/project was previously set as the root of the search.
+    In OBS the default root of the search is the project openSUSE.
```

Comment:
OBS is not build.opensuse.org, we should not mention specific instances here I think...

Reply:
I rephrased the description and also added some examples. Please have another look.